src/KingSystem/Damage/dmgStruct20.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <basis/seadTypes.h>
+#include <prim/seadRuntimeTypeInfo.h>
+
+#include "KingSystem/Utils/Types.h"
+
+namespace ksys::dmg {
+
+// What exactly Struct20 is, isn't known yet.
+// It is used by the Damage Managers, especially DamageManager and up, not so much in
+// DamageManagerBase. It contains flags and values which seems to be returned and modified when
+// handling damage.
+
+// TODO: Figure out exactly what Struct20 does and rename struct + variables to fit.
+
+class Struct20Base {
+    SEAD_RTTI_BASE(Struct20Base)
+
+public:
+    virtual ~Struct20Base(){};
+
+    virtual void reset();
+    virtual void combineMaybe(Struct20Base* other);
+};
+
+class Struct20 : public Struct20Base {
+    SEAD_RTTI_OVERRIDE(Struct20, Struct20Base)
+
+public:
+    ~Struct20() { return; };
```

Comment:
Assuming having a non-empty function body is necessary to match:
```suggestion
    ~Struct20() override { ; }
```

Otherwise, `~Struct20() override = default;` would be preferable

Reply:
With `~Struct20() override = default;` one of the destructors doesn't appear in the print_decomp_symbols any more, and cant match.